packages/conf-rdkit/conf-rdkit.0.1/opam

```diff
@@ -16,11 +16,10 @@ depexts: [
 ]
 post-messages:
     "This package requires rdkit to be installed. If you are using homebrew, you might want to try:
-    'brew tap rdkit/rdkit; brew install rdkit; pip3 install numpy; brew install --HEAD rdkit --with-python3 --with-inchi'"
+    'brew tap rdkit/rdkit; pip3 install numpy; brew install --HEAD rdkit --with-python3 --with-inchi'"
     {failure & os = "macos"}
 synopsis: "Virtual package relying on rdkit library installation"
 description: """
 This package can only install if the rdkit devel library is installed
 on the system."""
 extra-files: ["test.cpp" "md5=8079a0c593690540fbcb2f848f5b43d5"]
```

Comment:
Conf flag has to be there. The camelus warning is an opam bug that will be fixed soon